lib/components/track-changes/src/Insertion.ts

```diff
@@ -78,16 +78,20 @@ export class Insertion extends Component<InsertionProps, InsertionChild> {
 	public override async toNode(ancestry: ComponentAncestor[]): Promise<Node> {
 		return create(
 			`
-				element ${QNS.w}ins {
-					attribute ${QNS.w}id { $id },
-					attribute ${QNS.w}author { $author },
-					attribute ${QNS.w}date { $date },
-					$children
-				}
+				let $attrs := [
+					attribute ${QNS.w}id { $id }, 
+					if ($author) then attribute ${QNS.w}author { $author } else (),
+					if ($date) then attribute ${QNS.w}date { $date } else ()
+				]
+
+				return element ${QNS.w}ins { $attrs, $children }
 			`,
 			{
 				...this.props,
-				date: this.props.date?.toISOString(),
+				date: this.props.date
+					? new Date(this.props.date).toISOString()
```

Comment:
`this.props.date` is already a Date object, right? No need to create a new Date from that one.